app/experimenter/nimbus-ui/src/AppLayout/index.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
Oh no I missed that this doesn't have an MPL header! It's okay, I don't want to do another CI round. We can add it later.

Reply:
Dang it, I missed it too. I swear that needs to be automated. With whatever I'm doing next I'll add it here unless you beat me to it. :P